examples/advanced/sklearn-svm/jobs/sklearn_svm_base/app/custom/svm_learner.py

```diff
@@ -14,6 +14,7 @@
 
 from typing import Optional, Tuple
 
+from cuml.svm import SVC as SVC_gpu
```

Comment:
Do we want to depend on both cuml and sklearn here? Shouldn't it be an optional import?

Reply:
right, let me do the import inside

Reply:
updated